src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/entity/ActionPlanActivity.kt

```diff
@@ -0,0 +1,23 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity
+
+import java.time.OffsetDateTime
+import java.util.UUID
+import javax.persistence.Embeddable
+import javax.persistence.FetchType
+import javax.persistence.Id
+import javax.persistence.ManyToOne
+import javax.validation.constraints.NotNull
+
+@Embeddable
+data class ActionPlanActivity(
+
+  // Attributes
+  var description: String,
```

Comment:
can this be `val`?

Reply:
also, i notice this one doesn't have a `@NotNull` annotation on, but it's value is not nullable. 

i've been thinking for a while that the not null annotations aren't nessecary in kotlin for this reason, what do you think?

Reply:
Will change to val. In terms of @NotNull I've always been in favour of them when using the validation framework.